src/tests/utils/convert.js

```diff
@@ -70,19 +265,89 @@ describe('convert', () => {
     fs.removeSync(tempAppDir);
   });
 
-  it('should create separate files', async () => {
-    await convertApp(legacyApp, appDefinition, tempAppDir, true);
-    [
-      '.zapierapprc',
-      '.gitignore',
-      '.env',
-      'package.json',
-      'index.js',
-      'triggers/movie.js',
-      'test/triggers/movie.js'
-    ].forEach(filename => {
-      const filepath = path.join(tempAppDir, filename);
-      fs.existsSync(filepath).should.be.true();
+  describe('legacy apps', () => {
+    it('should create separate files', async () => {
+      await convertLegacyApp(legacyApp, legacyAppDefinition, tempAppDir, true);
+      [
+        '.zapierapprc',
+        '.gitignore',
+        '.env',
+        'package.json',
+        'index.js',
+        'triggers/movie.js',
+        'test/triggers/movie.js'
+      ].forEach(filename => {
+        const filepath = path.join(tempAppDir, filename);
+        fs.existsSync(filepath).should.be.true(`failed to create ${filename}`);
+      });
+    });
+  });
+
+  describe.only('visual apps apps', () => {
+    it('should create separate files', async () => {
```

Comment:
I'm wondering if we need tests to verify the output of files 🤔 For instance, to ensure we get dynamic inputs, functions are correctly written, files are ordered correctly, etc. 

Maybe overkill?

Reply:
nah, more tests are good! I can cover a couple of key cases in there.